README.md

```diff
@@ -1,6 +1,7 @@
 # ClimaCoupler.jl
 
-Coupler Specific Shared Development
+Provides coupled system time stepping control and support for mapping import and export
+boundary information between components.
```

Comment:
It would be good to emphasize more the difference between ClimaCoupler and the ClimaEarth experiment (and what's the supposed to be in the two respectively)